packages/typespec-autorest-canonical/src/types.ts

```diff
@@ -0,0 +1,591 @@
+import { ExtensionKey } from "@typespec/openapi";
```

Comment:
Is that going to be the exact same schema as `OpenAPI2 with autorest extensions`(schema for autorest emitter)? or will there be some different extensions?

Reply:
this is the very first version of the new emitter, and for now there is no different extensions. there may be more in the future.